annotations/views/relationtemplate_views.py

```diff
@@ -208,5 +206,13 @@ def create_from_relationtemplate(request, template_id):
     else:   # Not sure if we want to do anything for GET requests at this point.
         response_data = {}
 
-
     return JsonResponse(response_data)
+
+
+@staff_member_required
+def delete_relationtemplate(request, template_id):
+    if request.method == 'POST':
+        RelationTemplate.objects.filter(id=template_id).delete()
+        RelationTemplatePart.objects.filter(id=template_id).delete()
```

Comment:
this should be done in a transaction:
https://docs.djangoproject.com/en/1.11/topics/db/transactions/#controlling-transactions-explicitly

Reply:
Changed implementation of deletion logic to execute in an atomic transaction